mantidproject/mantid
# Indirect Data Reduction Interface, Symmetrise tab behaviour changes

**Description of work.**
Users were able to click the Run button before preview had been run, and preview was able to be ran before minimum or maximum, this causes errors because when preview is ran it sets some values for run. Therefore Run can only be clicked after preview has been, preview can only be clicked once the values for max or min has changed from their original values. This behaviour has been worked on between myself, @AnthonyLim23 and @SpencerHowells

**Report to:** Sanghamitra (ISIS)

**To test:**
- Change the instrument to IRIS
- Enter run 26173
- Change the detector grouping to individual
- Click Run

- Go to the symmetrise tab
- change it from file to workspace
- Move one of the min or max lines on the top graph to set the values.
- Click preview
- Click Run
- Notice how it runs fine



Fixes #25192. 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
